{{cookiecutter.repo_name}}/.github/workflows/gh-ci.yaml

```diff
@@ -0,0 +1,258 @@
+
+    steps:
+    - uses: actions/checkout@v3
+
+    - name: Set up Python
+      uses: actions/setup-python@v3
+      with:
+        python-version: 3.9
+
+    - name: Install Pylint
+      run: |
+        which pip
+        which python
+        pip install pylint
+
+    - name: Run Pylint
+      env:
+        PYLINTRC: .pylintrc
+      run: |
+        pylint {{ cookiecutter.repo_name }}
+
+
+  pypi_check:
+    if: "github.repository == '{{ cookiecutter.github_username }}/{{ cookiecutter.repo_name }}'"
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v3
+
+{% if cookiecutter.dependency_source == 'Dependencies from pip only (no conda)' %}
+    - name: Set up Python {{ '${{ matrix.python-version }}' }}
+      uses: actions/setup-python@v3
+      with:
+        python-version: {{ '${{ matrix.python-version }}' }}
+
+{% else %}
+    # More info on options: https://github.com/conda-incubator/setup-miniconda
+    - name: Install conda dependencies
+      uses: conda-incubator/setup-miniconda@v2
+      with:
+        python-version: {{ '${{ matrix.python-version }}' }}
+        add-pip-as-python-dependency: true
+        architecture: x64
+{% if cookiecutter.dependency_source == 'Prefer conda-forge over the default anaconda channel with pip fallback' %}
+        mamba-version: "*"
+        channels: conda-forge, defaults
+{% elif cookiecutter.dependency_source == 'Prefer default anaconda channel with pip fallback' %}
+        channels: defaults
+{% endif %}
+        auto-update-conda: true
+        show-channel-urls: true
+{% endif %}
+
+    - name: Install dependencies
+      run: |
+        pip install setuptools cython numpy twine
+
+    - name: Build package
+      run: |
+        python setup.py sdist
+
+    - name: Check package build
+      run: |
+        DISTRIBUTION=$(ls -t1 dist/{{ cookiecutter.repo_name }}-*.tar.gz | head -n 1)
+        test -n "${DISTRIBUTION}" || { echo "no distribution dist/{{ cookiecutter.repo_name }}-*.tar.gz found"; exit 1; }
+        echo "twine check $DISTRIBUTION"
+        twine check $DISTRIBUTION
+  
+  install_from_source_pip:
+    if: "github.repository == '{{ cookiecutter.github_username }}/{{ cookiecutter.repo_name }}'"
+    runs-on: ubuntu-latest
+    strategy:
+        fail-fast: false
+        matrix:
+          python-version: ["3.7", "3.8", "3.9"]
+
+    steps:
+    - uses: actions/checkout@v3
+
+    - name: Set up Python {{ '${{ matrix.python-version }}' }}
+      uses: actions/setup-python@v3
+      with:
+        python-version: {{ '${{ matrix.python-version }}' }}
+
+    - name: Install Poetry
+      uses: abatilo/actions-poetry@v2.1.3
+    
+    - name: Build from source
+      run: |
+        poetry install
+        poetry update
+        poetry show
+
+    - name: Check success
+      run: |
+        poetry run python -c "import MDAnalysis as mda; print(mda.__version__)"
+        PACKAGES=$(poetry show)
+        if [[ $PACKAGES != *"mdanalysis"* ]] ; then
```

Comment:
Is this a shell glob pattern? Wouldn't have thought that works with `!=`.

Reply:
Yes --
```
$ if [[ "asdfasdf" != *"g"* ]] ; then echo "asdfasdf has no g" ; else echo "asdfasdf has a g" ; fi
asdfasdf has no g
$ if [[ "asdfasdf" != *"f"* ]] ; then echo "asdfasdf has no f" ; else echo "asdfasdf has a f" ; fi
asdfasdf has a f
```

but it's probably redundant if the import works.